src/Duende.Bff/SessionManagement/SessionCleanupHost.cs

```diff
@@ -121,4 +126,13 @@ async Task RunAsync(CancellationToken cancellationToken = default)
             _logger.LogError("Exception deleting expired sessions: {exception}", ex.Message);
         }
     }
+
+    bool IsIUserSessionStoreCleanupRegistered()
```

Comment:
Not sure what net version you target, but if it's only 6+, there's supposedly a better way to check for registration (haven't used it myself yet).
See here https://andrewlock.net/exploring-dotnet-6-part-10-new-dependency-injection-features-in-dotnet-6/#detecting-if-a-service-is-registered-in-the-di-container

Reply:
Ah, nifty. Thanks for that.